src/main/java/iudx/resource/server/databroker/listeners/AsyncQueryListener.java

```diff
@@ -37,7 +38,7 @@ public void start() {
           if (startHandler.succeeded()) {
             LOGGER.trace("starting Q listener for Async query");
             client.basicConsumer(
-                ASYNC_QUERY_Q,
+                "rs-async-q-test",
```

Comment:
Revert back to actual queue name.

Reply:
Here's the commit where the change was done : [Update queue](https://github.com/datakaveri/iudx-resource-server/pull/425/commits/7d0373a054dfde8b2399833269ad1e0f7cee0da3)